src/eureka/S5_lightcurve_fitting/s5_fit.py

```diff
@@ -114,6 +120,48 @@ def fitlc(eventlabel, ecf_path=None, s4_meta=None, input_meta=None):
 
             # Load in the S4 metadata used for this particular aperture pair
             meta = load_specific_s4_meta_info(meta)
+            filename_S4_hold = meta.filename_S4_LCData.split(os.sep)[-1]
+            lc = xrio.readXR(meta.inputdir+os.sep+filename_S4_hold)
+
+            meta.mwhites_nexp = []
+            if meta.multwhite:
+                # Need to normalize each one if doing a joint fit
+                lc_whites = []
+
+                meta.stimes_plt = [lc.time.values[0]]
+                meta.etimes_plt = [lc.time.values[-1]]
+                meta.mwhites_nexp = [len(lc.time.values)]
+
+                lc_whites.append(lc)
+                for p in range(len(meta.inputdirlist)):
+                    # Specify where glob should search for the save file
+                    path = os.path.join(meta.topdir, meta.inputdirlist[p])
+                    # Search
+                    path = glob(path+os.sep+f'**{os.sep}*LCData.h5',
+                                recursive=True)
+                    if len(path) == 0:
+                        raise AssertionError(
+                            'Unable to find any LCData save files at '
+                            f'{path}')
+                    elif len(path) > 1:
+                        print(f'WARNING: Found {len(path)} LCData save '
+                              f'files... Using {path[0]}')
+                    # Use the first file found
+                    path = path[0]
+                    lc_hold = xrio.readXR(path)
+                    meta.wave_low = np.append(meta.wave_low,
+                                              lc_hold.wave_low.values)
+                    meta.wave_hi = np.append(meta.wave_hi,
+                                             lc_hold.wave_hi.values)
+
+                    meta.stimes_plt = np.append(meta.stimes_plt,
```

Comment:
I noticed those lines were added when I was reading through the code before submitting the PR and didn't recognize them. I think they're something Erin made and needed but are no longer used. If I figure out what they're used for I'' document them, otherwise I'll delete them if they're not used as I suspect.

Reply:
Ah, correction - I hadn't noticed mwhites_nexp in that list which is important (and I'll leave a comment in the code) - the others are indeed unused and will be deleted